org.eclipse.scout.rt.dataobject.test/src/main/java/org/eclipse/scout/rt/dataobject/id/AbstractIdCodecTest.java

```diff
@@ -762,7 +762,7 @@ public void testSignatureIdWithoutPassword() {
   }
 
```

Comment:
Add testcases for FixtureBooleanId

Reply:
Meaning specific test cases for IdCodec? Let's see how good the codec can deal with Date and Boolean

Reply:
Added them (also added boolean to IdCodec itself)